Explorer/Assets/DCL/WebRequests/Texture/GetTextureArguments.cs

```diff
@@ -1,12 +1,16 @@
+using Plugins.TexturesFuse.TexturesServerWrap.Unzips;
+
 namespace DCL.WebRequests
 {
     public readonly struct GetTextureArguments
     {
-        public readonly bool IsReadable;
+        public readonly ITexturesUnzip TexturesUnzip;
```

Comment:
So basically `TexturesUnzip` will go away

Reply:
Seems to be one of my comments disappeared:

Now you inject `ITexturesUnzip` per `TextureRequest`. However it does not change from texture to texture so it can be taken as a "global" dependency. 
I propose you to change it:

-instead of static initialization `internal static readonly InitializeRequest<GetTextureArguments, GetTextureWebRequest> GET_TEXTURE = GetTextureWebRequest.Initialize;` move it to `WebRequestController`
- thus, you will be able to pass `ITextureUnzip` to `WebRequestController` and propagate it respectively
- you won't need an injection of `IArgumentFactory` anymore in all consumers

Reply:
Not really, it could be changed in future because some of textures may require use different compression formats and options. I'd keep the current solution